_posts/2022-10-04-clicky-reflection-alm7468.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+author: alm7468
+title: “April’s Clicky Reflection”
+---
+
+I’m trying to do a simple I-Spy point-and-click game experience. 
+I created the intro setup where 4 options are presented. The user is prompted to double click one option (double clicking is a way to incorporate the required trinket screen click + user selection click). The screen should clear, the puzzle image becomes the background, and the user is prompted to click on a particular item. 
```

Comment:
This is a really cool interface/concept, and I am also wondering how to only keep the puzzle image in the background and get rid of the 4 rectangles once you enter the game

Reply:
Yea I am still working on it! But this is where I am at right now.